src/codeql.ts

```diff
@@ -230,6 +277,102 @@ async function getSourceLocationPrefix(codeql: string) {
   return resolvedDatabase.sourceLocationPrefix;
 }
 
+interface QueryPackRunResults {
+  queries: Array<{
+    queryPath: string;
+    relativeBqrsFilePath: string;
+    bqrsInfo: BQRSInfo;
+  }>;
+  totalResultsCount: number;
+  resultsBasePath: string;
+}
+
+async function getQueryPackRunResults(
```

Comment:
Minor point but I find some of the names a bit of a mouthful. Do they need to have `QueryPack` in the name or would `getRunResults` or `getQueryResults` be ok?

Reply:
There is a [`RunQueryResult`](https://github.com/github/codeql-variant-analysis-action/blob/cc327c42612adbc12416fd3bebf0c312caaeef44/src/codeql.ts#L13) so I wanted to differentiate from that. In general naming in this part of the codebase seems to have room for improvement, but I didn't want to make too many changes.